src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.scss

```diff
@@ -12,6 +12,19 @@
   cursor: pointer;
 }
 
+.edit-button {
+  color: #fff;
+  background-color: #38598a;
+  border-color: #38598a;  
+  font-size: 14px;
+  margin-right: 20px;
+
+  &:hover {
+    background-color: #2d476f;
+    border-color: #294266;
+  }
+}
```

Comment:
Added style so button matches BCGov standard. The mat-raised-button by default looks nothing like it's supposed to.